app/helpers/posts_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PostsHelper
```

Comment:
- [ ] Considering you were advised not to use the generate command, so that you can get a hands-on approach to how routing and controllers work. But you used them anyway. Kindly remove this generated file.